vault/identity_store.go

```diff
@@ -645,15 +644,7 @@ func (i *IdentityStore) initialize(ctx context.Context, req *logical.Initializat
 	if err := i.storeOIDCDefaultResources(ctx, req.Storage); err != nil {
 		return err
 	}
-
-	entry, err := logical.StorageEntryJSON(caseSensitivityKey, &casesensitivity{
-		DisableLowerCasedNames: i.disableLowerCasedNames,
-	})
-	if err != nil {
-		return err
-	}
-
-	return i.view.Put(ctx, entry)
+	return nil
```

Comment:
Should we delete the storage entry if it exists?

Reply:
I think upon upgrade, if the storage entry exists we can remove it as it is of no use anymore.